### DISCUSSION TITLE
Sync dictionary on multiple hosts

### DISCUSSION THREAD

**Discussion by 130s** [google/mozc#850] [Q&A]
Thanks for the great tool first of all (I've been using mozc for more than 5 years on Ubuntu, Android and OSX)!!

How do people keep the dictionary synchronized on multiple computers?

Since I do not find a feature in `mozc` to do that AFAICT, what I do is to use file sync tool (e.g. Dropbox) to keep the exported dict file. That works ok but only ok since it can get cumbersome whenever adding changes to the dict, as the dict is read in by the mozc is separate from the exported dict file I think and I'll have to manually update the dict on every host (I use more than 4 hosts on daily basis).

If this usecase makes sense, would that be considered as an enhancement?

**Comment by 130s** [comment]
I see this ticket labeled `obsolete` and closed, but as an active user of `mozc` the usecase I described https://github.com/google/mozc/issues/426#issue-270101560 is still valid but not realized yet AFAIK.
Thanks for the great tool and maintenance.

**Reply by yukawa** [reply]
We do realize such a use case, but realistically it's unlikely that we can implement such a feature in the near future due to our engineering resources, especially because maintaining existing features is already a challenge for us.

**Reply by 130s** [reply]
Understood, then, it's your discretion whether a not-planned feature request to remain open or to be closed (I prefer the former as it makes others finding it easier, but not much easier).

**Comment by hiroyuki-komatsu** [comment]
Thank you for the update.

I have reopened this issue and removed the `obsolete` label.
